Libraries/Network/RCTNetworking.ios.js

```diff
@@ -53,11 +52,11 @@ class RCTNetworking extends NativeEventEmitter {
   }
 
   abortRequest(requestId: number) {
-    RCTNetworkingNative.abortRequest(requestId);
+    NativeNetworking.abortRequest(requestId);
   }
 
   clearCookies(callback: (result: boolean) => any) {
```

Comment:
```callback: (result: boolean) => mixed```

Reply:
why mixed?

Reply:
Here is the reason [flow fry ...](https://flow.org/try/#0FDAmFMGMBsEMCdwAIBmBXAdpALgSwPYZICCAFJLNNAEayQDWAXEqQJQC8AfAG766itmvfgG4QwMmyRckAbwC+rMRNKwAzgE8sLVtM5zFIoA)

Reply:
Fair call, changed